src/serialization/CCLACSets.jl

```diff
@@ -0,0 +1,180 @@
+module CCLACSets
```

Comment:
Add a module-level docstring explaining what you told me in chat: why this module exists, some example use cases.